client/src/utils/api-util.js

```diff
@@ -132,4 +164,51 @@ export default async function uploadImagesToS3(images) {
     }
 
     return imageUrls;
+}
+
+export async function deleteImagesFromS3(imageUrls) {
+    // Setup the global AWS config
+    AWS.config.update({
+        region: 'us-east-2',
+        accessKeyId: `${process.env.REACT_APP_AWS_ACCESS_KEY_ID}`,
+        secretAccessKey: `${process.env.REACT_APP_AWS_SECRET_ACCESS_KEY}`
+    });
+
+    // Create a new AWS S3 Client
+    const s3Client = new AWS.S3();
+
+    // Setup parameters
+    const params = {
+        Bucket: s3ImageBucketName
+    };
+
+    for (const imageUrl of imageUrls) {
+        params['Key'] = getS3ImageObjectKeyFromObjectUrl(imageUrl);
+
+        try {
+            // Execute the delete operation
+            await s3Client.deleteObject(params).promise();
+        } catch (err) {
+            console.log("ERROR DELETING IMAGE FROM S3: " + imageUrl);
+            console.log("ERROR: " + err.message);
+        }
+    }
+}
+
+/**
+ * Helper function to get the S3 Object Key for an Image using the specified Object URL
+ * @param objectUrl - The Object URL to use to parse the Image Object Key
+ */
+function getS3ImageObjectKeyFromObjectUrl(objectUrl) {
+    const objectUrlArray = objectUrl.split('/');
+    return s3ImageBucketFolderName + objectUrlArray[objectUrlArray.length - 1];
+}
```

Comment:
One of the bugs I fixed was that we urlencode the key before storing it in the image table. You should make sure this still works after that's merged

Reply:
Going to double check that this and the other function still work now.

Reply:
Alright, made a fix to this function. Had to decode the URL before splitting it into an array. Going to check the other one now.